src/asciidoc/docs/guides-general/capture-user-input.adoc

```diff
@@ -0,0 +1,319 @@
+= Capturing user input
+:page-slug: capture-user-input
+:page-description: How to capture user input using `Koji.ui.capture` controls.
+:includespath: ../_includes
+:imagesdir: path-to-images-directory
+
+Koji provides a set of predefined user-input controls that you can use in your app's frontend to capture information and content from your users.
```

Comment:
We need to add some use cases that explain what this means. Use examples that are relevant to our primary messaging/use cases.

Reply:
The rest of the topic already explains this. This is the summary of the topic.

Reply:
Part of the challenge for developers right now is understanding what they should build on Koji and why. Providing some simple examples up front can help tell that story. It doesn’t have to be long. Just a sentence or two. “For example, your app might allow users to add photos to a social collage or …”

Reply:
Added.